### DISCUSSION TITLE
rEFInd+rEFInd-minimal主题进一步美化 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#68] [General]
# rEFInd+rEFInd-minimal主题进一步美化 - 夏沫花火zzz🌙 (Muska_Ami)のLife

设置指南rEFInd 显示比例问题修复我电脑屏幕比例 16:9 的，但是默认貌似当成 4:3 的了，通过 rEFInd 的 resolution 选项可以手动指定分辨率，举例1080P： 123456789101112131415161718# Set the screen's video resolution. Pass this option one of the following

https://blog.1l1.icu/2024/03/31/refind-refind-minimal-zhu-ti-jin-yi-bu-mei-hua/

**Comment by ltzXiaoYanMo** [comment]
:thinking:
